Framework/MDAlgorithms/src/ConvertToDiffractionMDWorkspace2.cpp

```diff
@@ -100,12 +104,13 @@ void ConvertToDiffractionMDWorkspace2::init() {
       "order to merge them later\n");
   setPropertyGroup("MinRecursionDepth", getBoxSettingsGroupName());
 
-  std::vector<double> extents{-50, +50};
+  std::vector<double> extents;
   declareProperty(
```

Comment:
I agree. I know it's a pain, but at least this way users can get the old behaviour back without installing an older version of Mantid. Try to share as much code as possible between the two version though.